gnucash/import-export/ofx/gnc-ofx-import.c

```diff
@@ -1038,6 +1039,54 @@ double ofx_get_investment_amount(const struct OfxTransactionData* data)
     }
 }
 
+// This callback is called when the user is done matching transactions.
+static void
+gnc_ofx_match_done (GtkDialog *dialog, gint response_id, gpointer user_data)
+{
+    ofx_info* info = (ofx_info*) user_data;
+    GSList *iter = NULL;
+    
+    if (info->statement == NULL)
+        return;
+    
+    if (response_id == GTK_RESPONSE_OK)
+    {
+        gint response = GTK_RESPONSE_NONE;
+        // Prepare a dialog to ask the user whether they want to open a reconcile window.
+        GtkWidget* dialog = gtk_message_dialog_new ( info->parent,
+                                                    GTK_DIALOG_MODAL | GTK_DIALOG_DESTROY_WITH_PARENT,
+                                                    GTK_MESSAGE_WARNING, GTK_BUTTONS_NONE,
+                                                    "%s", _("Your statement(s) include(s) balance information,\ndo you want to reconcile now?"));
+        // Remember the user's response in our prefs.
+        const char* warning = GNC_PREF_WARN_REC_AFTER_MATCH;
+        gtk_dialog_add_button (GTK_DIALOG(dialog),_("_NO"), GTK_RESPONSE_CANCEL);
+        gtk_dialog_add_button (GTK_DIALOG(dialog), _("_YES"), GTK_RESPONSE_ACCEPT);
+        // Open a reconcile window for each balance statement found.
+        for (iter=info->statement; iter; iter=iter->next)
+        {
+            struct OfxStatementData* statement = (struct OfxStatementData*) iter->data;
+            Account* account = gnc_import_select_account (gnc_gen_trans_list_widget(gnc_ofx_importer_gui),
+                                                          statement->account_id,
+                                                          0, NULL, NULL, ACCT_TYPE_NONE, NULL, NULL);
+            if (account && statement->ledger_balance_valid)
+            {
+                // Now's the time to ask the user (but only once)
+                if (response == GTK_RESPONSE_NONE)
+                {
+                    response =  gnc_dialog_run (GTK_DIALOG(dialog), warning);
+                    if (response != GTK_RESPONSE_ACCEPT)
+                        break;
+                }
+                gnc_numeric value = double_to_gnc_numeric (statement->ledger_balance, xaccAccountGetCommoditySCU (account), GNC_HOW_RND_ROUND_HALF_UP);
+                recnWindowWithBalance (GTK_WIDGET (info->parent), account, value, statement->ledger_balance_date);
```

Comment:
This is the travis failure, add 'gnc_numeric value;' to the start of the if block and change this line to 'value = ...'